examples/pt2/torch_compile_openvino/stable_diffusion/requirements.txt

```diff
@@ -0,0 +1 @@
+diffusers==0.27.2
```

Comment:
Can we change this to  `>=0.27.2`

We have dependabot alerts which will keep flagging this otherwise